src/common/function/src/scalars/json/json_path_exists.rs

```diff
@@ -64,38 +64,40 @@ impl Function for JsonPathExistsFunction {
         let paths = &columns[1];
 
         let size = jsons.len();
-        let datatype = jsons.data_type();
         let mut results = BooleanVectorBuilder::with_capacity(size);
 
-        match datatype {
-            // JSON data type uses binary vector
-            ConcreteDataType::Binary(_) => {
-                for i in 0..size {
-                    let json = jsons.get_ref(i);
-                    let path = paths.get_ref(i);
+        for i in 0..size {
+            let json = jsons.get_ref(i);
+            let path = paths.get_ref(i);
 
+            match json.data_type() {
+                ConcreteDataType::Binary(_) => {
```

Comment:
We should use `jsons.data_type()` because the data type of `ValueRef::Null` isn't `Binary`. But we can match `json` and handle `ValueRef::Binary` and `ValueRef::Null`.

Reply:
I see your point, but I’m curious about how we would handle cases where some items in the array are `ValueRef::Null` while others are not. Or, all items' datatype are supposed to be the same?

Reply:
Okay, I'll update the code.